lib/start.js

```diff
@@ -3,18 +3,77 @@
 const { buildRuntime, loadConfig } = require('@platformatic/runtime')
 const { join } = require('path')
 const { parseArgs } = require('util')
+const { request } = require('undici')
+const { exec } = require('child_process')
+const { promisify } = require('util')
+const closeWithGrace = require('close-with-grace')
+
```

Comment:
The changes in this file needs some tests

Reply:
d8e9fc9